code/game/objects/items/stacks/sheets/runed_metal.dm

```diff
@@ -0,0 +1,114 @@
+		title = "altar", \
+		result_type = /obj/structure/destructible/cult/item_dispenser/altar, \
+		req_amount = 3, \
+		time = 4 SECONDS, \
+		one_per_turf = TRUE, \
+		on_floor = TRUE, \
+		desc = span_cultbold("Altar: Can make Eldritch Whetstones, Construct Shells, and Flasks of Unholy Water."), \
+		required_noun = "runed metal sheet", \
+	), \
+	new /datum/stack_recipe/radial( \
+		title = "archives", \
+		result_type = /obj/structure/destructible/cult/item_dispenser/archives, \
+		req_amount = 3, \
+		time = 4 SECONDS, \
+		one_per_turf = TRUE, \
+		on_floor = TRUE, \
+		desc = span_cultbold("Archives: Can make Zealot's Blindfolds, Shuttle Curse Orbs, \
+			and Veil Walker equipment. Emits Light."), \
+		required_noun = "runed metal sheet", \
+	), \
+	new /datum/stack_recipe/radial( \
+		title = "daemon forge", \
+		result_type = /obj/structure/destructible/cult/item_dispenser/forge, \
+		req_amount = 3, \
+		time = 4 SECONDS, \
+		one_per_turf = TRUE, \
+		on_floor = TRUE, \
+		desc = span_cultbold("Daemon Forge: Can make Nar'Sien Hardened Armor, Flagellant's Robes, \
+			and Eldritch Longswords. Emits Light."), \
+		required_noun = "runed metal sheet", \
+	), \
+	new /datum/stack_recipe/radial( \
+		title = "runed door", \
+		result_type = /obj/machinery/door/airlock/cult, \
+		time = 5 SECONDS, \
+		one_per_turf = TRUE, \
+		on_floor = TRUE, \
+		desc = span_cultbold("Runed Door: A weak door which stuns non-blood cultists who touch it."), \
+		required_noun = "runed metal sheet", \
+	), \
+	new /datum/stack_recipe/radial( \
+		title = "runed girder", \
+		result_type = /obj/structure/girder/cult, \
+		time = 5 SECONDS, \
+		one_per_turf = TRUE, \
+		on_floor = TRUE, \
+		desc = span_cultbold("Runed Girder: A weak girder that can be instantly destroyed by ritual daggers. Not a recommended usage of runed metal."), \
+		required_noun = "runed metal sheet", \
+	), \
+))
+
+/obj/item/stack/sheet/runed_metal
+	name = "runed metal"
+	desc = "Sheets of cold metal with shifting inscriptions writ upon them."
+	singular_name = "runed metal sheet"
+	icon_state = "sheet-runed"
+	inhand_icon_state = "sheet-runed"
+	icon = 'icons/obj/stack_objects.dmi'
+	mats_per_unit = list(/datum/material/runedmetal = MINERAL_MATERIAL_AMOUNT)
+	sheettype = "runed"
+	merge_type = /obj/item/stack/sheet/runed_metal
+	novariants = TRUE
+	grind_results = list(/datum/reagent/iron = 5, /datum/reagent/blood = 15)
+	material_type = /datum/material/runedmetal
+	has_unique_girder = TRUE
+	use_radial = TRUE
+
+/obj/item/stack/sheet/runed_metal/interact(mob/user)
+	if(!IS_CULTIST(user))
+		to_chat(user, span_warning("Only one with forbidden knowledge could hope to work this metal..."))
+		return FALSE
+
+	var/turf/user_turf = get_turf(user)
+	var/area/user_area = get_area(user)
+
+	var/is_valid_turf = user_turf && (is_station_level(user_turf.z) || is_mining_level(user_turf.z))
+	var/is_valid_area = user_area && (user_area.area_flags & CULT_PERMITTED)
```

Comment:
is the first check even necessary? area_flags includes both levels

Reply:
In certain events where a cult permitted area ends up in deep space, such as survival pods and shuttles

Reply:
i didnt even know survival pods were allowed, i wonder what the reason behind it is. not for this pr though